docker/Dockerfile

```diff
@@ -53,6 +55,18 @@ RUN set -e; \
 
 RUN set -e; \
     export DEB_MAX_ATTEMPT=10; \
+    export CRIS_IMG_ARCH="$(set -e; \
+        case $(uname -m) in \
+        'amd64' | 'x86_64' | 'x64') \
+            echo 'amd64'; \
+            ;; \
+        'arm64' | 'aarch64') \
+            echo 'arm64'; \
```

Comment:
Let's call it aarch64 instead, amd64/arm64 is too confusing.

Reply:
I prefer to be consistent with `TARGETARCH`, while it gives you `amd64/arm64`

Reply:
I won't be surprised if later on someone (including editor's auto-completion) makes related mistakes on this.

Reply:
Me too, it is a terrible choice but hopefully our CI will catch the typos.

There seems to be a de-facto standard here to call them arm64/amd64, so I don't want to reinvent other names.